README.md

```
@@ -182,10 +187,29 @@ function App() {
+本项目包含以下移动端优化：
+
+- **响应式CSS**: 移动优先断点 (320px, 480px, 767px)
+- **Flexbox布局**: 按钮在移动端垂直堆叠，桌面端水平排列
+- **触摸目标**: 所有按钮最小 44px × 44px
+- **WCAG无障碍**: 对比度 ≥4.5:1，支持键盘导航 (focus-visible)
+- **触摸设备**: 通过 `@media (hover: hover)` 隐藏悬停效果
+- **横屏模式**: 自动调整布局避免重叠
+
+### CSS文件结构
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)